tests/lib/config/config-validator.js

```diff
@@ -443,18 +443,6 @@ describe("Validator", () => {
             assert.throws(fn, "tests:\n\tConfiguration for rule \"mock-rule\" is invalid:\n\tSeverity should be one of the following: 0 = off, 1 = warn, 2 = error (you passed '[ \"error\" ]').\n");
         });
 
-        it("should only check warning level for nonexistent rules", () => {
```

Comment:
I think that we should change the tests with the new behavior instead of removal.